setup.py

```diff
@@ -70,6 +71,24 @@ def read(*names, **kwargs):
         return fp.read()
 
 
+def KaolinCUDAExtension(*args, **kwargs):
+    FLAGS = ['-Wno-deprecated-declarations']
+    kwargs = copy.deepcopy(kwargs)
+    if 'extra_compile_args' in kwargs:
+        kwargs['extra_compile_args'] += FLAGS
+    else:
+        kwargs['extra_compile_args'] = FLAGS
+    return CUDAExtension(*args, **kwargs)
+
+
+class KaolinBuildExtension(BuildExtension):
+    def build_extensions(self):
+        FLAG_BLACKLIST = ['-Wstrict-prototypes']
```

Comment:
Is this a common flag to be blacklisted ? Otherwise we can just fix the function declarations ?

Reply:
Filtering -Wstrict-prototypes is to silence these

`cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++`